### PR TITLE
Removed outdated Bitbucket screen shots

### PR DESCRIPTION
**Pull request by tamigoodall**
Removed outdated Bitbucket screen shots and simplifed text and headers into steps. Removed TOC = True because now the doc has fewer headings.

### CODE REVIEW

File: articles/connections/social/bitbucket.md [hunk 1]
```diff
@@ -14,85 +12,49 @@ useCase:
...
+6. Copy your *Client Id* and *Client Secret* into Auth0. In a separate window, log in to your [Auth0 Dashboard](${manage_url}) and select **Connections > Social** in the left nav. Select **Bitbucket**:
 
 ![](/media/articles/connections/social/bitbucket/bitbucket-05.png)
 
-Copy the `Key` and `Secret` from the **OAuth integrated applications** page on Bitbucket into the fields on this page on Auth0.
-
-Select the **Permissions** you want to enable.
-
-Click **SAVE**.
+7. Copy the `Key` and `Secret` from the **OAuth integrated applications** page on Bitbucket into the fields on this page on Auth0. Select the **Permissions** you want to enable. Click **SAVE**.
```
Comment (jeffreylees) [inline articles/connections/social/bitbucket.md:39]: You changed the case of key and secret above, same here?

**Review by jeffreylees — COMMENTED** [review]
Are you replacing the removed screenshots, or is that unnecessary?

File: articles/connections/social/bitbucket.md (hunk 1 above)
Comment (tamigoodall) [reply articles/connections/social/bitbucket.md:39]: Good catch. I changed it back so they all match (title case).

Comment (tamigoodall) [reply articles/connections/social/bitbucket.md:39]: I'm not replacing the screenshots. There were superfluous and didn't add much at all. I kept the relevant ones (that hadn't changed). This way, if Bitbucket modifies their UI in the future, we won't have to chase updates.

**Review by jeffreylees — APPROVED** [review]
Sounds good to me

**Pull request merged**